### ISSUE TITLE
Docker building error

### ISSUE THREAD

**Issue by Akanichi** [borisdayma/dalle-mini#260]
When i started building docker image i get this error message, i think we have to change Docker file to solve this issue, i will look on the subject and update this issue.
```
Collecting typing_extensions
  Downloading typing_extensions-4.2.0-py3-none-any.whl (24 kB)
ERROR: Could not find a version that satisfies the requirement jaxlib==0.3.10+cuda11.cudnn82 (from jax[cuda]) (from versions: 0.1.32, 0.1.40, 0.1.41, 0.1.42, 0.1.43, 0.1.44, 0.1.46, 0.1.50, 0.1.51, 0.1.52, 0.1.55, 0.1.56, 0.1.57, 0.1.58, 0.1.59, 0.1.60, 0.1.61, 0.1.62, 0.1.63, 0.1.64, 0.1.65, 0.1.66, 0.1.67, 0.1.68, 0.1.69, 0.1.70, 0.1.71, 0.1.72, 0.1.73, 0.1.74, 0.1.75, 0.1.76, 0.3.0, 0.3.2, 0.3.5, 0.3.7, 0.3.8, 0.3.10)
ERROR: No matching distribution found for jaxlib==0.3.10+cuda11.cudnn82 (from jax[cuda])
The command '/bin/sh -c pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_releases.html   && pip install -q   git+https://github.com/borisdayma/dalle-mini.git   git+https://github.com/patil-suraj/vqgan-jax.git' returned a non-zero code: 1

```

**Comment by Akanichi** [comment]
There is already a solution here, it's not an old error.
https://github.com/google/jax/discussions/11074

**Comment by Akanichi** [comment]
wasn't able to solve even with last python version, there are other error that occur when i did it.

**Comment by cdgamedev** [comment]
I got it to work by changing the docker file:

```
RUN pip install https://storage.googleapis.com/jax-releases/nocuda/jaxlib-0.3.10-cp38-none-manylinux2014_x86_64.whl
```

this removes its lookup for jax and instead gets the exact version needed for python3.8

**Comment by wacky6** [comment]
Looks like Jax team splits the release file into two.

This one is for the CUDA Jax: https://storage.googleapis.com/jax-releases/jax_cuda_releases.html

I can build the image by replacing the URL in Dockerfile

**Comment by borisdayma** [comment]
Should now be fixed through #292
